tenant-deployments/tenant-role-assignment/main.bicep

```diff
@@ -0,0 +1,17 @@
+targetScope = 'tenant'
+
+@description('principalId if the user that will be given contributor access to the resourceGroup')
```

Comment:
This should be the tenant?

Reply:
Ah yes - I copied that from the old one  :) Fixed.